test/controllers/users/users.signup.spec.js

```diff
@@ -0,0 +1,122 @@
+const request = require('supertest');
+const app = require('../../../app');
+const UserModel = require('../../../app/models').user;
+
+describe('POST /users/signup', () => {
+  it('Should create a user', async () => {
+    const { statusCode, body } = await request(app)
+      .post('/users/signup')
+      .send({
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'jdoe@wolox.com.ar',
+        password: 'Sherman33'
+      });
+
+    const createdUser = await UserModel.findOne({
+      where: {
+        email: 'jdoe@wolox.com.ar'
+      }
+    });
+
+    expect(statusCode).toEqual(201);
+    expect(body.token).toBeDefined();
+    expect(body.user).toBeDefined();
+    expect(createdUser.firstName).toEqual('John');
+    expect(createdUser.lastName).toEqual('Doe');
+    expect(createdUser.email).toEqual('jdoe@wolox.com.ar');
+  });
```

Comment:
Not as correction, but as additional information:
Sometimes we prefer to use an structure like the following, to get a better failing tracking (every expect has it own track)
```js
describe('some tests', () => {
  let response;
  let createdUser;

  beforeAll(async () => {
    response = await request(app)
      .post('/users/signup')
      .send({ firstName: 'John', lastName: 'Doe', email: 'jdoe@wolox.com.ar', password: 'Sherman33' });
    createdUser = await UserModel.findOne({ where: { email: 'jdoe@wolox.com.ar' } });
  });

  it('should response 201', () => {
    expect(statusCode).toEqual(201);
  });
  it('should have property token', () => {
    expect(body.token).toBeDefined();
  });
  it('should have property user', () => {
    expect(body.user).toBeDefined();
  });
});
```